features/remote-poller-gwt/src/main/resources/org/opennms/features/poller/remote/gwt/public/app.jsp

```diff
@@ -31,11 +31,36 @@
 
 <%@page language="java" contentType="text/html; charset=UTF-8" %>
 <%@page import="java.net.URLEncoder"%>
+<%@ page import="org.apache.commons.lang.StringEscapeUtils" %>
+
+<%!
+	public static boolean isValidConfiguration() {
+		return isUrlValid() && isAttributionValid();
+	}
+
+	public static boolean isUrlValid() {
+		return isDefined(System.getProperty("gwt.openlayers.url"));
+	}
+
+	public static boolean isAttributionValid() {
+		return isDefined(System.getProperty("gwt.openlayers.options.attribution"));
+	}
+
+	public static boolean isDefined(String input) {
+		return input != null && !"".equals(input);
+	}
+%>
 <%
+
   String mapImplementation = System.getProperty("gwt.maptype", "");
-  String openlayersUrl = System.getProperty("gwt.openlayers.url", "http://otile1.mqcdn.com/tiles/1.0.0/osm/${z}/${x}/${y}.png");
+  String openlayersUrl = System.getProperty("gwt.openlayers.url", "");
```

Comment:
Shouldn't we make this default to the tiles.opennms.org URL rather than giving broken maps on startup?